changelog.d/62.doc.md

```diff
@@ -0,0 +1 @@
+Add obtain_token management command
```

Comment:
Could you please rename the file to `62.misc.md`? After all, you are adding new functionality.